resources/styleheets/main.css

```diff
@@ -0,0 +1,276 @@
+body {
+  padding: 0;
+  margin: 0;
+  height: 100%;
+  width: 100%;
+  background-color: #1B1717;
+  font-family: "Almarai", sans-serif;
+}
+
+.container {
+  height: 100%;
+  display: flex;
+  color: #ccc;
+}
+
+h1, h2, h3, h4, h5, h6 {
+  margin: 0;
+}
+
```

Comment:
this have to be in _base.scss
check out the [7 pattern standarts ](https://sass-guidelin.es/#naming-conventions)
to know the content that goes inside each folder